src/scalib_ext/ranklib/src/histogram.rs

```diff
@@ -7,11 +7,17 @@ mod bnp {
     include!(concat!(env!("OUT_DIR"), "/binding_bnp.rs"));
 }
 
+const CONV_LIMIT: f64 = 34359738368.0 as f64; //define a certain limit
+
 pub trait Histogram {
     fn new(size: usize) -> Self;
+    fn type_str(&self) -> &'static str;
+    fn rescale(&self) -> Self;
```

Comment:
This function seems internal to the `F64Hist` implementation and doesn't need to be in the trait.